site/en/setup.md

```diff
@@ -166,23 +189,43 @@ made in containers back to your local Docker image.)
 
 ### Testing the Docker installation
 
-To start, download and extract a document from the [OCR-D GT Repo](https://ocr-d-repo.scc.kit.edu/api/v1/metastore/bagit/):
+To start, download and extract a document from the [OCR-D GT Repo](https://ola-hd.ocr-d.de/search?q=&fulltextsearch=false&metadatasearch=false&isGT=true&perPageRecords=30):
 
 ```sh
-wget 'https://ocr-d-repo.scc.kit.edu/api/v1/dataresources/736a2f9a-92c6-4fe3-a457-edfa3eab1fe3/data/wundt_grundriss_1896.ocrd.zip'
-unzip wundt_grundriss_1896.ocrd.zip
+wget "https://ola-hd.ocr-d.de/api/export?id=21.T11998/0000-001C-F82E-8&internalId=false" -O wundt_grundriss_1896.ocrd.zip
+sudo unzip wundt_grundriss_1896.ocrd.zip
 cd data
 ```
 
-Let's segment the images in file group `OCR-D-IMG` from the zip file into regions, thereby creating a 
-[PAGE-XML](https://github.com/PRImA-Research-Lab/PAGE-XML) file group `OCR-D-SEG-BLOCK-DOCKER`)
+Now, spin up the docker container:
 
-You can spin up a Docker container, mounting the current working directory like this:
+```sh
+docker run --user $(id -u) --workdir /data --volume $PWD:/data -it ocrd/all bash
+```
+
+Your command line should start with something similar to:
 
 ```sh
-docker run --user $(id -u) --workdir /data --volume $PWD:/data -- ocrd/all:maximum ocrd-tesserocr-segment-region -I OCR-D-IMG -O OCR-D-SEG-BLOCK-DOCKER
+I have no name!@ade9a4692fcd:/data$
 ```
 
+After spinning up the container, you can use the installation and call the processors the same way as in the native installation.
+
+Alternatively, you would have to [translate each command to a docker call](/en/user_guide#translating-native-commands-to-docker-calls) (not recommended).
+
+Let's segment the images in file group `OCR-D-IMG` from the zip file into regions, thereby creating a 
+[PAGE-XML](https://github.com/PRImA-Research-Lab/PAGE-XML) file group `OCR-D-SEG-BLOCK-DOCKER`):
+
+```sh
+ocrd-tesserocr-segment-region -I OCR-D-IMG -O OCR-D-SEG-BLOCK-DOCKER
```

Comment:
Will this work without downloading models and mounting them in the docker call, i.e. a fresh ocrd_all installation? Otherwise we might want to use a different example here.

Reply:
Yes, it will. We install en.traineddata by default, which is the "model" needed (to be present) for layout analysis.